Tests/FunctionalTests/TraitTests.swift

```diff
@@ -405,13 +403,15 @@ struct TraitTests {
                 extraArgs: [
                     "--enable-all-traits",
                     "--disable-default-traits",
-                    "--experimental-prune-unused-dependencies",
                 ],
                 buildSystem: buildSystem,
             )
             // We expect no warnings to be produced. Specifically no unused dependency warnings.
             let unusedDependencyRegex = try Regex("warning: '.*': dependency '.*' is not used by any target")
             #expect(!stderr.contains(unusedDependencyRegex))
+            if buildSystem == .swiftbuild {
+                print(stderr)
```

Comment:
question: out of curiosity, why are we printing `stderr` here?  if it's meant to troubleshoot a failed expectation, we can add a comment to the `#expect(...)` or `#require(...)`

See the [expect(_:_:sourcelocation:) API docs](https://developer.apple.com/documentation/testing/expect(_:_:sourcelocation:)) for details.